tools/release-scripts/test.js

```diff
@@ -16,10 +9,10 @@ function lint() {
 }
 
 function testAndLint() {
-  return Promise.all([
-    test(),
-    lint()
-  ]);
+  console.log('Running: '.cyan + 'eslint and tests'.green);
+
+  return exec('npm run test')
+    .then(() => console.log('Completed: '.cyan + 'eslint and tests'.green));
```

Comment:
Addressed in https://github.com/react-bootstrap/react-bootstrap/pull/865#issuecomment-113793672